jobs/JEVS_MESOSCALE_STATS

```diff
@@ -41,6 +41,7 @@ export DATA=${DATA:-${DATAROOT:?}/${jobid:?}}
 mkdir -p $DATA
 cd $DATA
 
+
```

Comment:
@ShelleyMelchior-NOAA Before this PR is merged, can you please remove this extra space added in this PR?